datalad_container/containers_remove.py

```diff
@@ -0,0 +1,99 @@
+
+__docformat__ = 'restructuredtext'
+
+import logging
+import os.path as op
+
+from datalad.interface.base import Interface
+from datalad.interface.base import build_doc
+from datalad.support.param import Parameter
+from datalad.distribution.dataset import datasetmethod, EnsureDataset
+from datalad.distribution.dataset import require_dataset
+from datalad.interface.utils import eval_results
+from datalad.support.constraints import EnsureNone
+from datalad.support.constraints import EnsureStr
+from datalad.interface.results import get_status_dict
+
+# required bound methods
+from datalad.coreapi import save
+from datalad.coreapi import remove
+
+
+lgr = logging.getLogger("datalad.containers.containers_remove")
+
+
+@build_doc
+# all commands must be derived from Interface
+class ContainersRemove(Interface):
+    # first docstring line is used a short description in the cmdline help
+    # the rest is put in the verbose help and manpage
+    """Remove a known container from a dataset
+    """
+
+    # parameters of the command, must be exhaustive
+    _params_ = dict(
+        dataset=Parameter(
+            args=("-d", "--dataset"),
+            doc="""specify the dataset to query. If no dataset is given, an
+            attempt is made to identify the dataset based on the current
+            working directory""",
+            constraints=EnsureDataset() | EnsureNone()),
+        label=Parameter(
```

Comment:
Shouldn't this be "name" for consistency with add/run?

Reply:
Yes, but I actually want to swap that in `add` too. So consistency will be restored.